pkg/config/app_config.go

```diff
@@ -236,15 +236,17 @@ confirmOnQuit: false
 // AppState stores data between runs of the app like when the last update check
 // was performed and which other repos have been checked out
 type AppState struct {
-	LastUpdateCheck int64
-	RecentRepos     []string
+	LastUpdateCheck    int64
+	RecentRepos        []string
+	RecentPrivateRepos []string
 }
 
 func getDefaultAppState() []byte {
 	return []byte(`
-  lastUpdateCheck: 0
-  recentRepos: []
-`)
+    lastUpdateCheck: 0
+    recentRepos: []
+    RecentPrivateRepos: []
```

Comment:
to line up with the other existing variables we have in the app state, this should be in camelCase
```suggestion
    recentPrivateRepos: []
```

Reply:
I'll do that